.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
I think we should put os-specific stuff like this in users' global gitignore files rather than in the repo.

Reply:
For sure, this is just cra boilerplate. I'll remove it!